docs/modules/chalk.md

```diff
@@ -0,0 +1,35 @@
+# chalk
+
+`chalk` and similar libraries have various alternatives which are lighter and faster.
+
+# Alternatives
+
+## picocolors
+
+A widely used alternative which is much lighter and faster.
+
+[Project Page](https://github.com/alexeyraspopov/picocolors)
+
+[npm](https://npmjs.com/package/picocolors)
+
+## ansis
+
+A lighter alternative which also supports 256 color creation.
+
+[Project Page](https://github.com/webdiscus/ansis)
+
+[npm](https://npmjs.com/package/ansis)
+
+## styleText (Node 20.x and above)
+
+Node itself has a `styleText` function in the built-in `util` library.
+
+You can use it like so:
+
+```ts
```

Comment:
is it really necessary for it to be typescript? extremely small nit

```suggestion
```js
```

Reply:
It's a habit more than anything just in case I end up writing types. Happy to change it